UI.Tests/WinCheckerTests.cs

```diff
@@ -0,0 +1,405 @@
+using UI.Enums;
+using UI.Entities;
+using FluentAssertions;
+
+namespace UI.Tests;
+
+public class WinCheckerTests
+{
+    #region checking rows
+
+    [Fact]
+    public void HasWon_WhenThreeMatchingMarksInFirstRow_ShouldReturnTrue()
+    {
+        // arrange
+        Player host = new()
+        {
+          ConnectionId = "player one connection hub id",
+          Name = "Dan",
+          ImageUrl = "https://www.pngall.com/wp-content/uploads/12/Avatar-Profile-Vector.png",
+          Mark = Marks.X,
+          HasTurn = true
+        };
+        string GameId = "g00";
+
+        Marks markToCheck = Marks.X;
+        Marks[] gameBoard = new Marks[] { Marks.X, Marks.X, Marks.X, Marks.X, Marks.X, Marks.O, Marks.X, Marks.X, Marks.X };
+
+        // act
+        Game winChecker = new Game(GameId, host);
+        bool hasWon = winChecker.HasWon(markToCheck, gameBoard);

```

Comment:
Make `HasWon()` take the `Move` model. That already includes the `Mark` and also, the `Game` object already has access to the game board inside itself, so no need to pass that to it.